backend/isoko-backend/src/handlers/business/get-search-results.js

```diff
@@ -1,19 +1,99 @@
+const dynamodb = require('aws-sdk/clients/dynamodb');
+const _ = require('lodash');
+const docClient = new dynamodb.DocumentClient();
+const { BUSINESS_TABLE } = require('../../constants');
+
+const processLocationParam = (locationParam) => {
```

Comment:
I'm a little worried that this function might have some issues with a few particular cities. (Although, I'm not super confident about this, I just wan't to put it out there in case anybody else knows)

Some problematic cities could be Missouri/St. Louis, Missouri/Lee's Summit, or Illinois/Bloomington-Normal I'm not a URL expert at all, but special characters like ' or . or -  in a city might be a cause for concern in URL's.

Reply:
Yeah you make a good point, I'm going to move the location parameter into the body of the request so we don't run into that issue anymore. Good looks Cole